Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.2
```

Comment:
if we increase the version here, every importing package must also be on swift 5.2 afaik. I'm not sure if we want to drop support for 5.1 yet?

Reply:
Fair enough! I'll change that back

Reply:
I think we still need the other changes in `Package.swift` when building with Xcode 11.4 though